operator/Makefile

```
@@ -201,7 +201,14 @@ GET_GITHUB_RELEASE_FN = get_github_release() { \
 	[ -x $${1} ] || { \
 		set -euxo pipefail ;\
 		mkdir -p bin ;\
-		curl --fail --location --output $${1} $${2} ;\
+		for i in {1..5}; do \
+			set +e ;\
+			curl --fail --location --output $${1} $${2} ;\
+			code=$$? ;\
+			set -e ;\
+			[ $$code -eq 0 ] && break ;\
```

Let's simplify this bit as:
```suggestion
			curl --fail --location --output $${1} $${2} && break ;\
```